source/backend/apimodels/Models/Concepts/DispositionFile/DispositionSalePurchaserAgentModel.cs

```diff
@@ -15,7 +15,7 @@ public class DispositionSalePurchaserAgentModel : BaseConcurrentModel
         /// <summary>
         /// Parent Disposition Sale.
         /// </summary>
-        public long DispositionSaleId { get; set; }
+        public long? DispositionSaleId { get; set; }
```

Comment:
does this match the ef required field?

Reply:
For new entities I am passing null in the model and letting EF to resolve the primary key to be generated.

Reply:
if the ef model does not allow nulls, then the mapper will assign a zero to it. It is probably better to have that being explicit on the api model too

Reply:
updated